app/chitter_app.rb

```diff
@@ -0,0 +1,75 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  set :session_secret, 'super secret'
+  register Sinatra::Flash
+  use Rack::MethodOverride
+
+  get '/' do
+    redirect '/posts'
+  end
+
+  get '/posts' do
```

Comment:
use `peeps` as the routes - not `posts`; 'posts' is not part of the language of the domain.